test/jit/test_backend_nnapi.py

```diff
@@ -29,73 +28,49 @@
-    TODO: After Nnapi delegate is finished T91991928,
-    add tests for running the module (currently only lowers the model)
-    """
+class TestNnapiBackend(TestNNAPI):
     def setUp(self):
         super().setUp()
-        # Load nnapi delegate library
-        torch_root = Path(__file__).resolve().parent.parent.parent
-        p = torch_root / 'build' / 'lib' / 'libnnapi_backend.so'
-        torch.ops.load_library(str(p))
-
-        # Nnapi set up
-        # Avoid saturation in fbgemm
-        torch.backends.quantized.engine = 'qnnpack'
 
-        libneuralnetworks_path = os.environ.get("LIBNEURALNETWORKS_PATH")
-        if libneuralnetworks_path:
-            ctypes.cdll.LoadLibrary(libneuralnetworks_path)
-            print("Will attempt to run NNAPI models.")
-            self.can_run_nnapi = True
-        else:
-            self.can_run_nnapi = False
-
-    def test_execution(self):
         # Save default dtype
         module = torch.nn.PReLU()
-        default_dtype = module.weight.dtype
+        self.default_dtype = module.weight.dtype
         # Change dtype to float32 (since a different unit test changed dtype to float64,
         # which is not supported by the Android NNAPI delegate)
         # Float32 should typically be the default in other files.
         torch.set_default_dtype(torch.float32)
-        module = torch.nn.PReLU()
-        args = torch.tensor([[1.0, -1.0, 2.0, -2.0]]).unsqueeze(-1).unsqueeze(-1)
 
-        # Trace and lower PreLu module
-        traced = torch.jit.trace(module, args)
-        compile_spec = {"forward": {"inputs": args}}
-        nnapi_model = torch._C._jit_to_backend("nnapi", traced, compile_spec)
+        # Load nnapi delegate library
+        torch_root = Path(__file__).resolve().parent.parent.parent
+        p = torch_root / 'build' / 'lib' / 'libnnapi_backend.so'
+        torch.ops.load_library(str(p))
 
-        # Change dtype back to the default
-        torch.set_default_dtype(default_dtype)
+        # Disable execution tests, only test lowering modules
+        # TODO: Re-enable execution tests after the Nnapi delegate is complete
+        super().set_can_run_nnapi(False)
 
-# First skip is needed for IS_WINDOWS or IS_MACOS to skip the tests.
-# Second skip is because ASAN is currently causing an error.
-# It is still unclear how to resolve this. T95764916
-@unittest.skipIf(TEST_WITH_ROCM or IS_SANDCASTLE or IS_WINDOWS or IS_MACOS or IS_FBCODE,
-                 "Non-portable load_library call used in test")
-@unittest.skipIf(TEST_WITH_ASAN, "Unresolved bug with ASAN")
-class TestNnapiBackend(JitTestCase):
-    """
-    This class wraps all Nnapi test classes
-    TODO: Add more comprehensive Nnapi tests (currently only a basic one)
-    """
-    def __init__(self, name):
-        super().__init__(name)
-        self.prelu_test = NnapiBackendPReLUTest(name)
+    # Override
+    def call_convert_to_nnapi(self, traced_module, args):
```

Comment:
Could you change this function name to "test_delegate_to_nnapi", to differentiate it from the original convert method?

Reply:
I'm using this method to override one in test_nnapi.py, so I don't think that name works because in test_nnapi.py the method doesn't use the delegate. Maybe "test_lowering_to_nnapi" is better?

Reply:
Oh, the name can't start with test or else the TestCase will try to run the function without arguments. How about "call_lowering_to_nnapi"?